common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -172,46 +174,60 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
             const matchingCanvas = index && canvases[index];
             return (
               <ListItem key={i}>
-                <ListLink
-                  style={{ textDecoration: 'none', cursor: 'pointer' }}
-                  onClick={() => {
-                    if (index) {
-                      setIsMobileSidebarActive(false);
-                      setActiveIndex(index || 0);
-                      mainViewerRef &&
-                        mainViewerRef.current &&
-                        mainViewerRef.current.scrollToItem(index || 0, 'start');
-                    }
-                  }}
+                <NextLink
+                  {...itemLink(
+                    {
+                      canvas: index || 0,
```

Comment:
I think we `1` index `canvas`?

Reply:
The canvas is 0 index but what is displayed to the user is 1 indexed

Reply:
😕